packages/ui/src/app/pages/WorkingGroups/components/WorkingGroupsTabs.tsx

```diff
@@ -1,15 +1,23 @@
 import React from 'react'
 
-import { Tabs } from '../../../../common/components/Tabs'
-import { usePageTabs } from '../../../hooks/usePageTabs'
+import { Tabs } from '@/common/components/Tabs'
+import { useMyMemberships } from '@/memberships/hooks/useMyMemberships'
+
+import { TabsDefinition, usePageTabs } from '../../../hooks/usePageTabs'
+
+const basicTabs: TabsDefinition[] = [
+  ['Openings', '/working-groups/openings'],
+  ['Working Groups', '/working-groups'],
+]
+const memberTabs: TabsDefinition[] = [
+  ['My Applications', '/working-groups/my-applications'],
+  ['My Roles', '/working-groups/my-roles'],
+]
 
 export const WorkingGroupsTabs = () => {
-  const tabs = usePageTabs([
-    ['Openings', '/working-groups/openings'],
-    ['Working Groups', '/working-groups'],
-    ['My Applications', '/working-groups/my-applications'],
-    ['My Roles', '/working-groups/my-roles'],
-  ])
+  const { members } = useMyMemberships()
+  const avaliableTabs = members.length == 0 ? basicTabs : basicTabs.concat(memberTabs)
```

Comment:
WDYT about adding `hasMembers` to the hook?

I've found one other place that we check for this now, but I think it might be more.